posthog/hogql_queries/test/test_query_runner.py

```diff
@@ -113,6 +123,24 @@ def test_cache_payload(self):
                 "useMaterializedViews": True,
                 "usePresortedEventsTable": False,
             },
+            "data_warehouse_config": {
+                "revenue_analytics": {
+                    "base_currency": "GBP",
+                    "events": [
+                        {
+                            "couponProperty": "coupon",
+                            "currencyAwareDecimal": False,
+                            "eventName": "purchase",
+                            "productProperty": "product",
+                            "revenueCurrencyProperty": {
+                                "property": "currency",
+                                "static": None,
+                            },
+                            "revenueProperty": "revenue",
+                        }
+                    ],
```

Comment:
How often do/can these fields update?

Reply:
As in add new configs (keys) or change the values? New keys, very rarely, new values, everytime someone wants to add a new event source which shouldnt be very common, probably once ever? Why the concern?

Reply:
Once in a blue moon, really. There's not much reason to come back and change this after you've set it up - and it'll usually be set up when a company is small. There might be a cache busting storm when we release Rev Analytics to GA, so we need to be careful with that